edsnlp/processing/simple.py

```diff
@@ -56,26 +71,32 @@ def _flatten(list_of_lists: List[List[Any]]):
 def _pipe_generator(
     note: pd.DataFrame,
     nlp: Language,
+    context: List[str] = [],
     additional_spans: Union[List[str], str] = "discarded",
     extensions: ExtensionSchema = [],
-    batch_size: int = 1000,
+    batch_size: int = 50,
```

Comment:
Should this be part of the PR ?

Reply:
It isn't related but it still seems that 1000 is way too big, especially for medical texts which are usually pretty long